data_structures/src/mainnet_validations.rs

```diff
@@ -90,7 +90,7 @@ impl TapiEngine {
                         bit_counter.votes += 1;
                     }
                     if (epoch_to_update - bit_counter.init) % bit_counter.period == 0 {
-                        if (bit_counter.votes * 100) / bit_counter.period >= 80 {
+                        if (bit_counter.votes * 100) / bit_counter.period >= 10 {
```

Comment:
Dont forget to remove this after testing

Reply:
If all the tests pass when that line changes, it means that that line is not tested :eyes: